### ISSUE TITLE
New BULK DAIL Evaluator for Recoupment Dails

### ISSUE THREAD

**Issue by C-Love** [Hennepin-County/MAXIS-scripts#253]
There are two DAILS:
 - Recoupment has increased
 - Recoupment ended - no one responsible

These would evaluate for if the recoupment matches the current approved version by running it through background. Then it could remove the ones that are already done and leaving only actionable work.

**Comment by C-Love** [comment]
Initial idea from Aaron:

I’d probably have it start by asking if the new result was approved yet or not:
Yes - script proceeds to next dialog
No, new recoupment not approved yet – script instructs to review and approve results and run again.
No, recoupment is the same as before – script informs no further action needed, review to make sure the new recoupment was already approved and noted and delete the DAIL if appropriate.

Next section if Yes:
Dialog for “other notes”, the script reads the benefit amount and new recoupment amount and case notes the DAIL received and approval.


We think the script could actually run the case through background and check the recoupment vs the approved amount AND we should BULK it out.

We could separately create a DAIL Scrubber for HSRs that would call Eligibility Summary once it is done.

**Comment by courtrightd** [comment]
Upon review of scope, this would remove less than 40 dails of 100,000. Will not fix.
